frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -573,6 +580,15 @@ private static void validateUnmanagedBranchNotification(IGitMacheteRepositorySna
     }
   }
 
+  @Override
+  public void enableEnqueuingUpdates() {
+    ENQUEUING_UPDATES_DISABLED.set(false);
```

Comment:
I'd always call `queueRepositoryUpdateAndModelRefresh` once the updates are re-enabled